elasticmagic/expression.py

```diff
@@ -454,6 +454,9 @@ def missing(self, **kwargs):
     def in_(self, terms, **kwargs):
         return Terms(self, terms, **kwargs)
 
+    def not_in_(self, terms, **kwargs):
```

Comment:
but this is not sqlalchemy)
`not_in(..)` would look and read better than `notin_()` or `not_in_()`.